src/components/DynamicFormik/DynamicFormik.js

```diff
@@ -524,49 +524,49 @@ export default function DynamicFormik({
    * Get values from formik and normalize keys
    */
 
-  const normalizedInputs = inputs.map((input) => ({
-    ...input,
-    key: input.key.replace(/\./g, '||'),
-    requiredForKey:
-      typeof input.requiredForKey === 'string' ? input.requiredForKey.replace(/\./g, '||') : null,
-  }));
-
-  const normalizeValues = (values) => {
-    if (!Boolean(values)) return {};
-    let inputKeys = Object.entries(values);
-    let newValues = {};
-    inputKeys.forEach((value) => (newValues[value[0].replace(/\./g, '||')] = value[1]));
-    return newValues;
-  };
+  // const normalizedInputs = inputs.map((input) => ({
```

Comment:
Cane we remove this code. Otherwise looks nice and simple to me. We can add this to our end of year list to go through and make sure this works in all of our apps